pom.xml

```diff
@@ -1104,6 +1104,12 @@
                 <groupId>io.delta</groupId>
                 <artifactId>delta-core_${scala.binary.version}</artifactId>
                 <version>${delta.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
https://github.com/delta-io/delta/pull/676

Reply:
It's better to add some comments at this code place or add a `todo` to remind update this in future ?

Reply:
updated